Tests/Command/CleanMediaCommandTest.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Command;
+
+use Sonata\MediaBundle\Command\CleanMediaCommand;
+use Sonata\MediaBundle\Filesystem\Local;
+use Sonata\MediaBundle\Model\MediaManagerInterface;
+use Sonata\MediaBundle\Provider\Pool;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
+
+class CleanMediaCommandTest extends FilesystemTestCase
+{
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @var Application
+     */
+    protected $application;
+
+    /**
+     * @var ContainerAwareCommand
+     */
+    protected $command;
+
+    /**
+     * @var CommandTester
+     */
+    protected $tester;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Pool
+     */
+    private $pool;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|MediaManagerInterface
+     */
+    private $mediaManager;
+
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject|Local
+     */
+    private $fileSystemLocal;
```

Comment:
Can you create a ``BaseCommandTest`` or something like this, i think this would reduce duplicate code. what do you think?